plugin/completion/flags_manager.py

```diff
@@ -190,10 +205,33 @@ def get_flags(self, separate_includes):
         if self._clang_complete_file.was_modified():
             log.debug(" .clang_complete modified. Load new flags.")
             self._flags = FlagsManager.flags_from_clang_file(
-                self._clang_complete_file, separate_includes)
+                self._clang_complete_file)
 
         # the flags are now in final state, we can return them
-        return self._flags
+        return FlagsManager.merge_flags(self._initial_flags, list(self._flags))
```

Comment:
This will merge flags each time this method is called thus causing duplicated flags get added.
(On Windows that can also cause error when command line is too long but that is a separate issue)